qiskit_ibm_runtime/estimator.py

```diff
@@ -99,32 +92,12 @@ class Estimator(BaseEstimator):
 
     def __init__(
         self,
-        circuits: Optional[Union[QuantumCircuit, Iterable[QuantumCircuit]]] = None,
-        observables: Optional[Iterable[SparsePauliOp]] = None,
-        parameters: Optional[Iterable[Iterable[Parameter]]] = None,
-        service: Optional[QiskitRuntimeService] = None,
         session: Optional[Union[Session, str, IBMBackend]] = None,
```

Comment:
IMO, these options should be keyword only arguments. But it might be preference.